src/Sylius/Component/Core/Model/Taxonomy.php

```diff
@@ -12,17 +12,16 @@
 namespace Sylius\Component\Core\Model;
 
 use Sylius\Component\Taxonomy\Model\Taxonomy as BaseTaxonomy;
+use Sylius\Component\Taxonomy\Model\TaxonInterface as BaseTaxon;
```

Comment:
I had a look at whole file, and it's used only in constructor that just proxies parent constructor, so if we remove that constructor, we can remove this `use` statement and less code is better code :)